Sources/Datadog/Tracing/Utils/URLFilter.swift

```diff
@@ -40,6 +40,9 @@ internal struct URLFilter: URLFiltering, Equatable {
     /// matches hosts and their subdomains: example.com -> example.com, api.example.com, sub.example.com, etc.
     private static func buildRegexString(from hosts: Set<String>) -> String {
         return hosts.map {
+            if $0 == "." {
+                return "."
+            }
```

Comment:
We can't allow wildcard to be passed from the public API. Our approach for requests auto instrumentation, is to receive an exact list of hosts from the user, without generalisations, so we must find an other way around for tests instrumentation.